pytorch_lightning/trainer/supporters.py

```diff
@@ -120,26 +120,106 @@ def mean(self):
 
 
 class PredictionCollection(object):
-    def __init__(self, global_rank: int, world_size: int):
+
+    ID_KEY = 'id'
```

Comment:
Maybe some information as to what this is used for might be beneficial

Reply:
Done !